client/my-sites/post-type-list/post-action-counts/index.jsx

```diff
@@ -79,6 +81,35 @@ class PostActionCounts extends PureComponent {
 		);
 	}
 
+	renderViewCount() {
+		const { viewCount: count, numberFormat, postId, showViews, siteSlug, translate } = this.props;
+		if ( ! count || count < 1 || ! showViews ) {
+			return null;
+		}
+		const visibleText = translate( '%(count)s Recent View', '%(count)s Recent Views', {
+			count,
+			args: { count: numberFormat( count ) },
+		} );
+		const screenReaderText = translate( 'in the past 30 days', {
+			comment:
+				'specifies time period for recent view count, i.e. "10 Recent Views in the past 30 days."',
+		} );
+		const linkTitleText = `${ visibleText } ${ screenReaderText }`;
```

Comment:
Just realized breaking up a translation string like this is not kosher. Will update.